examples/sanitizers/ThreadSanitizerApp/BUILD

```diff
@@ -0,0 +1,49 @@
+load(
+    "@com_github_buildbuddy_io_rules_xcodeproj//xcodeproj:defs.bzl",
+    "xcodeproj",
+)
+load("//:xcodeproj_targets.bzl", "XCODEPROJ_TARGETS")
+load("@build_bazel_rules_apple//apple:ios.bzl", "ios_application")
+load("@build_bazel_rules_swift//swift:swift.bzl", "swift_library")
+load(
+    "//:xcodeproj_targets.bzl",
+    "IOS_BUNDLE_ID",
+)
+load("@build_bazel_rules_apple//apple:versioning.bzl", "apple_bundle_version")
+
+apple_bundle_version(
+    name = "Version",
+    build_version = "123.4.5",
+    short_version_string = "123.4",
+    visibility = ["//:__subpackages__"],
+)
+
+ios_application(
+    name = "ThreadSanitizerApp",
+    bundle_id = IOS_BUNDLE_ID,
```

Comment:
`IOS_BUNDLE_ID` was a variable mainly to help with provisioning profiles, so we can remove this and hardcode the same as others: `"io.buildbuddy.example"`.

Reply:
Cool! Updated